source/features/highlight-deleted-and-added-files-in-diffs.tsx

```diff
@@ -0,0 +1,69 @@
+import select from 'select-dom';
+import oneTime from 'onetime';
+import {observe} from 'selector-observer';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {observeOneMutation} from '../helpers/simplified-element-observer';
+
+async function loadDeferred(jumpList: Element): Promise<void> {
+	const loadJumpList = (jumpList: Element) => jumpList.parentElement!.dispatchEvent(new MouseEvent('mouseover'));
+	loadJumpList(jumpList);
+	// The event listener might not have been attached yet, so we can try twice
+	setTimeout(loadJumpList, 1000, jumpList);
+	await observeOneMutation(jumpList);
+}
+
+async function init(): Promise<void> {
+	const fileList = await elementReady([
+		'.toc-select details-menu', // `isPr`
```

Comment:
Consistent capitalization everywhere
```suggestion
		'.toc-select details-menu', // `isPR`
```